tests/thread_flood/tests/test_thread.py

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/python
```

Comment:
minor thing: for the sake of continuity, how about we rename this file to `01-run.py` to be consistent?

Reply:
Also: change to `#!/usr/bin/env python3` (as my 'old' mint has trouble with its old 2.7.x...), thx!